packages/client/components/ThreadedCommentBase.tsx

```diff
@@ -88,6 +90,20 @@ const ThreadedCommentBase = (props: Props) => {
     })
   }
 
+  useEffect(() => {
+    if (createdByUserNullable?.id === PARABOL_AI_USER_ID) {
+      const contentState = editorState.getCurrentContent()
+      const summary = contentState.getPlainText()
+      SendClientSegmentEventMutation(atmosphere, 'AI Summary Viewed', {
+        source: 'Discussion',
+        tier: viewer.tier,
+        meetingId,
+        discussionTopicId,
```

Comment:
@tghanken I added the `discussionTopicId` to the Discussion event as the `AI Summary Viewed` event will be triggered for every Discuss Stage. With `discussionTopicId`, we can see which topic is being summarised